system/CodeIgniter.php

```diff
@@ -147,6 +152,17 @@ class CodeIgniter
 	 */
 	public function __construct(App $config)
 	{
+		if (version_compare(PHP_VERSION, self::MIN_PHP_VERSION, '<'))
+		{
+			die(
+				sprintf(
+					'Your PHP version must be %s or higher to run CodeIgniter. Current version: %s',
```

Comment:
Is it possible, if this is added to Lang?

Reply:
@totoprayogo1916 if you mean multi for multi language, I am not sure since it early executed.

Reply:
`lang()` is definitely not available yet. In general I'm okay with strictly internal messages being in English. I understand there are some internationalization efforts for PHP core but I think this is still pretty common?